src/custom-elements/stencil.config.ts

```diff
@@ -128,6 +128,22 @@ export const config: Config = {
           else if (a.name < b.name) return -1
           return 0
         });
+
+        docs['components'].map(component => {
+          const checkIfPrivate = ['props', 'methods', 'events'];
+
+          Array.prototype.forEach.call(
+            checkIfPrivate,
+            (propertyTypeToCheck) => {
+              const titleToCheck = propertyTypeToCheck === 'events' ? 'event' : 'name';
+
+              component[propertyTypeToCheck] = component[propertyTypeToCheck]
+                .filter(propertyToCheck => {
+                  return propertyToCheck[titleToCheck].split('')[0] !== '_';
```

Comment:
Good one! All methods and properties preceded by underscore are meant for internal use only and for that reason they are not documented, right? :)

Reply:
Yes, it will filter out all the props, methods and events starting with underscore ;)